src/Taboo/TabooRoom.py

```diff
@@ -1,18 +1,25 @@
 """Taboo room"""
+import re
 
 from fwk.GamePlugin import GamePlugin
+from fwk.Trace import trace, Level
```

Comment:
expanded imports please. new additions later don't end up touching these existing lines that way

Reply:
ok. Done